lib/Dialect/Torch/Transforms/DecomposeComplexOps.cpp

```diff
@@ -4434,13 +4436,6 @@ class DecomposeAtenEmptyStridedOp
   LogicalResult matchAndRewrite(AtenEmptyStridedOp op,
                                 PatternRewriter &rewriter) const override {
     SmallVector<int64_t> sizeListInts, strideListInts;
-    if (!matchPattern(op.getSize(), m_TorchListOfConstantInts(sizeListInts)))
```

Comment:
These are necessary. The checks below expect `strideListInts` and `sizeListInts` to be populated